spec/oas3.patched/openapi/public/components/schemas/InterconnectionPort.yaml

```diff
@@ -1,36 +1,3 @@
-properties:
-  id:
-    format: uuid
-    type: string
-  organization:
-    $ref: './Href.yaml'
-  role:
-    description: Either 'primary' or 'secondary'.
-    type: string
-    enum:
-    - primary
-    - secondary
-  status:
-    type: string
-    description: For both Fabric VCs and Dedicated Ports, this will be 'requested' on creation and 'deleting' on deletion. Once the Fabric VC has found its corresponding Fabric connection,
-      this will turn to 'active'. For Dedicated Ports, once the dedicated port is associated, this will also turn to 'active'. For Fabric VCs, this can turn into an 'expired' state if the 
-      service token associated is expired.
-    enum:
-    - requested
-    - active
-    - deleting
-    - expired
-  switch_id:
-    description: A switch 'short ID'
-    type: string
-  virtual_circuits:
```

Comment:
I think it would be more correct here to patch the `VirtualCircuitList.yaml` file to specify a list schema:

```yaml
items:
  anyOf:
  - $ref: './VirtualCircuit.yaml'
  - $ref: './VrfVirtualCircuit.yaml'
type: array
```

Reply:
Looking through the schema more, I see there's a lot of precedent for the current structure of `VirtualCircuitList`, and changing that would require a bigger patch, so I'm on board with what you have here.

One thing to note is that we're working on a patch to address #58, and that will likely include a name change for the `VirtualCircuit` model.